dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -725,6 +741,31 @@ public Boolean checkWDL(File content){
         return false;
     }
 
+    private void verify(List<String> args) {
+        if (isAdmin) {
+            if (containsHelpRequest(args)) {
+                verifyHelp();
+            } else if (!args.isEmpty()) {
+                String entry = reqVal(args, "--entry");
+                String verifySource = optVal(args, "--verified-source", null);
+                final boolean unverifyRequest = isUnverifyRequest(args);
+                final boolean isScript = SCRIPT.get();
+                handleVerifyUnverify(entry, verifySource, unverifyRequest, isScript);
+            }
+        } else {
+            out("This command is only accessible to Admins.");
+        }
+    }
+
+    private static boolean isUnverifyRequest(List<String> args) {
```

Comment:
Try a lambda expression to simplify this, say https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#anyMatch-java.util.function.Predicate-